src/body/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Styled } from './styled';
+import { MoviesBoard } from './movies-board';
+
+export const Body = () => {
+    return (
+        <Styled.Body>                        
```

Comment:
What you'll have another page and you'll need use Body again ?
If you want reuse that component you have to make it:
<Styled.Body>
   {props.children}
</Styled.Body>